assets/js/apps/project_detail_view/components/ProjectPage/ProjectPage.jsx

```diff
@@ -0,0 +1,70 @@
+import React, { Component, useState, useEffect } from "react";
+import "./ProjectPage.css";
+import ExperimentList from "../ExperimentList/ExperimentList";
+import { Alert, Spinner } from "react-bootstrap";
+
+export default function ProjectPage(props) {
+
+  let [projectData, setProjectData] = useState(null);
+  let [responseStatus, setResponseStatus] = useState(false);
+
+  // fetches project data, sets state
+  useEffect(() => {
+    console.log("From use effect.");
+    fetch(`/api/v1/project/?id=${props.projectId}`, {
+      method: 'get',
+      headers: {
+        'Accept': 'application/json',
+        'Content-Type': 'application/json'
+      }
+    }).then(response => {
+      console.log(response)
+      setResponseStatus(response.status)
+      if (!response.ok) {
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 12 spaces but found 6.](https://app.codacy.com/gh/mytardis/mytardis/pullRequest?prid=7933948)
```suggestion
            if (!response.ok) {
```